Lit Labs Event Announcement (#979)
lit/lit.dev: packages/lit-dev-content/site/blog/2022-10-13-lit-labs-event.md

---
tags: blogPosts
layout: blog-post.html
title: "Get the low-down on Lit Labs"
summary: "We are having a virtual event on October 24th 2022 at 10 AM PST to talk about the status of Lit Labs!"
date: 2022-10-13
author:
  - elliott-marquez
eventDate: "Monday October 24th 2022 at 10AM PDT"

> ytLink: https://www.youtube.com/

Author: Yeah! I should ping @graynorton

---

> The Lit team is hosting a virtual event on **{{eventDate}}** to talk about the status of Lit Labs. Join us for detailed presentations on the current and future offerings of Lit labs, their statuses, and how you can help get them across the line and shape the future of Lit!

Reviewer: Nit: My initial reaction to "talk about the status of Lit Labs" sounded sorta ominous, along the lines of "let's talk about the status of our relationship" as if something bad is about to drop. 

Maybe something a little more obviously positive, like

```suggestion
The Lit team is hosting a virtual event on **{{eventDate}}** to talk about all of the exciting work going on in Lit Labs. Join us for detailed presentations on the current and future offerings of our Labs packages, their statuses, and how you can help get them across the line and shape the future of Lit!
```

> The Lit team is hosting a virtual event on **{{eventDate}}** to talk about all of the exciting work going on in Lit Labs. Join us for detailed presentations on the current and future offerings of our Labs packages, their statuses, and how you can help get them across the line and shape the future of Lit!

Reviewer 2: ```suggestion
The Lit team is hosting a virtual event on **{{eventDate}}** to talk about all of the exciting work going on in Lit Labs. We'll give an overview of Lit Labs, followed by detailed presentations on each of our Labs packages. And we'll talk about how you can help get these packages across the finish line and shape the future of Lit!
```

> The event will be hosted [**at this link**]({{ytLink}}) on YouTube. Make sure to sign up for reminders and notifications so you don't miss it!

Reviewer 2: Or alternately, make the whole thing a CTA (btw, I don't see a "reminders" option when I look at future livestream events... I see "Notify me," "Subscribe," and "Save" ...

```suggestion
To make sure you don't miss it, [**visit the live stream**]({{ytLink}}) and sign up for notifications. 
```

> The event will be hosted [**at this link**]({{ytLink}}) on YouTube. Make sure to sign up for reminders and notifications so you don't miss it!

Reviewer 2: I'm not sure I have the right words. "visit the live stream" seems a little weird. "visit the YouTube event page"? idk

<img src="/images/blog/labs-event/lit-labs.svg" width="720" class="block centered">

## What is Lit Labs?

[Lit Labs]({{site.baseurl}}/docs/libraries/labs/) is an umbrella for projects related to Lit that aren't quite ready for production. These projects may be experimental or incomplete, or simply looking for real-world feedback. Lit Labs projects are published under the [`@lit-labs` npm scope](https://www.npmjs.com/search?q=%40lit-labs).

## What will be covered at the Lit Labs event?

> The Lit team will be covering information on how individual Labs projects can help you in your Web Component projects as well as their current status for Labs graduation into stable. Here is a list of the projects that will be covered:

Reviewer 2: ```suggestion
The Lit team will be covering information on how individual Labs projects can help you in your Web Component projects as well as their current status for Labs graduation into stable. Here are the projects we'll be covering:
```

- [Task](https://github.com/lit/lit/tree/main/packages/labs/task)
- [Virtualizer](https://github.com/lit/lit/tree/main/packages/labs/virtualizer)
- [Motion](https://github.com/lit/lit/tree/main/packages/labs/motion)
- [Context](https://github.com/lit/lit/tree/main/packages/labs/context)
- [React](https://github.com/lit/lit/tree/main/packages/labs/react)
- [CLI](https://github.com/lit/lit/tree/main/packages/labs/cli)-generated Framework Wrappers
- [SSR](https://github.com/lit/lit/tree/main/packages/labs/ssr)
- [Observers](https://github.com/lit/lit/tree/main/packages/labs/observers)

## When is it?

> The event will be on **{{eventDate}}**. Make sure to sign up for YouTube reminders and notifications [**at this link**]({{ytLink}}) so you don't miss it!

Reviewer 2: As above ...  "at this link" ==> "on the event page", "for the live stream event," or something like that.
